admin/config.js

```diff
@@ -174,7 +174,6 @@ module.exports = {
     newCheckWindow: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_NEW_CHECK_WINDOW') ? toBool(process.env.FEATURE_TOGGLE_NEW_CHECK_WINDOW) : true,
     pupilEdit: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_PUPIL_EDIT') ? toBool(process.env.FEATURE_TOGGLE_PUPIL_EDIT) : true,
     pupilRegisterV2: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_PUPIL_REGISTER_V2') ? toBool(process.env.FEATURE_TOGGLE_PUPIL_REGISTER_V2) : false,
-    schoolHomeViewV2: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_SCHOOL_HOME_VIEW_V2') ? toBool(process.env.FEATURE_TOGGLE_SCHOOL_HOME_VIEW_V2) : false,
-    _2020Mode: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_2020_MODE') ? toBool(process.env.FEATURE_TOGGLE_2020_MODE) : true
+    schoolHomeViewV2: {}.hasOwnProperty.call(process.env, 'FEATURE_TOGGLE_SCHOOL_HOME_VIEW_V2') ? toBool(process.env.FEATURE_TOGGLE_SCHOOL_HOME_VIEW_V2) : false
```

Comment:
i believe George has set this to true by default in another PR, as its ready for use.